app/inbound/process-submitted-settlement.js

```diff
@@ -0,0 +1,25 @@
+const db = require('../data')
+const saveSettlement = require('./save-settlement')
+const getExistingSettlement = require('./get-existing-settlement')
+
+const processSubmittedSettlement = async (settlement) => {
+  console.log(settlement)
+  const transaction = await db.sequelize.transaction()
+
+  try {
+    const invoiceNumber = settlement.invoiceNumber
+    const existingSettlement = await getExistingSettlement(invoiceNumber, transaction)
```

Comment:
Need to tweak this a bit.  Because we're paying quarterly, we will receive a separate settlement for each quarterly payment with the same invoice number.

If we leave this as is, then we'll ignore quarters 2-4.

If you look in the `ffc-pay-processing` service and then `settlements.js`, we might be able to do something like that for a duplicate check.

Reply:
resolved by comparing `invoiceNumber` and `value` for each settlement